azure/table_azure_storage_container.go

```diff
@@ -0,0 +1,183 @@
+			KeyColumns:        plugin.AllColumns([]string{"name", "resource_group", "account_name"}),
+			Hydrate:           getStorageContainer,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "ContainerNotFound"}),
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listStorageAccounts,
+			Hydrate:       listStorageContainers,
+		},
+
+		Columns: []*plugin.Column{
+			// Basic info
+			{
+				Name:        "name",
+				Description: "The friendly name that identifies the container.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The container ID",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "type",
+				Description: "Specifies the type of the container.",
+				Type:        proto.ColumnType_STRING,
+			},
+			// Other details
+			{
+				Name:        "account_name",
+				Description: "The friendly name that identifies the storage account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID").Transform(idToAccountName),
+			},
+			{
+				Name:        "deleted",
+				Description: "Indicates whether the blob container was deleted.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("ContainerProperties.Deleted"),
+			},
+			{
+				Name:        "default_encryption_scope",
+				Description: "Default the container to use specified encryption scope for all writes.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContainerProperties.DefaultEncryptionScope"),
+			},
+			{
+				Name:        "public_access",
+				Description: "Specifies whether data in the container may be accessed publicly and the level of access.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContainerProperties.PublicAccess"),
+			},
+			{
+				Name:        "remaining_retention_days",
+				Description: "Remaining retention days for soft deleted blob container.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("ContainerProperties.RemainingRetentionDays"),
+			},
+			{
+				Name:        "version",
+				Description: "The version of the deleted blob container.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContainerProperties.Version"),
+			},
+			{
+				Name:        "container_properties",
+				Description: "The blob container properties.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ContainerProperties"),
+			},
+
+			// Standard steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Etag"),
```

Comment:
`Etag` is a system generated hash required for concurrency control, when we make an update or delete operation.
Remove `tags` column if it is not supported.

Reply:
removed